### DISCUSSION TITLE
Empty rows added when exporting to csv and excel

### DISCUSSION THREAD

**Discussion by EvaldasBurlingis** [yajra/laravel-datatables#2600] [Q&A]
Hi, 

I have a problem with exporting data to csv and excel. When exporting I keep getting two empty rows added to document and I think that messes up excel export. 

Here are images of exports:

![csv](https://user-images.githubusercontent.com/32436432/113095685-930fbb80-91fc-11eb-9a32-463d28b2a715.png)
![excel](https://user-images.githubusercontent.com/32436432/113095691-95721580-91fc-11eb-8963-f1cc786c2685.png)

The data that reach `buildExcelFile()` is good, with no empty array items.

Any idea what could be the cause of this problem?

**Comment by yajra** [comment]
What is your package version for dataTables and laravel-excel?

Anyways, I think this is timeout issue and the laravel dump where added on export. Maybe try updating to latest version, I made some improvements on the export process several days ago.
